src/clip.rs

```diff
@@ -29,10 +29,9 @@ pub fn copy_to_clipboard(s: &SafeString) -> Result<(), ()> {
 
 // On UNIX, the most stable way to copy to the clipboard is using one of the existing
 // and battle tested tools: xsel and xclip.
-#[cfg(all(unix, not(macos)))]
+#[cfg(all(unix, not(macos), not(windows)))]
 pub fn copy_to_clipboard(s: &SafeString) -> Result<(), ()> {
     use shell_escape;
-    use quale::which;
```

Comment:
This is useless, given the above comment, and breaks the UNIX builds (Linux, Mac OSX). Please revert this change, unless this is useful.

Reply:
Crap that shouldn't have been there. I'll fix that.

Reply:
Alright, thanks a lot ! :)